lnwallet/channel.go

```diff
@@ -1489,11 +1499,11 @@ func (lc *LightningChannel) ReceiveRevocation(revMsg *lnwire.CommitRevocation) (
 	// Now that we've verified the revocation update the state of the HTLC
 	// log as we may be able to prune portions of it now, and update their
 	// balance.
-	var htlcsToForward []*PaymentDescriptor
+	var includedHtlcs []*PaymentDescriptor
```

Comment:
I changed the name for several reasons:
1. For developers who don't know the internal mechanisms of working of  `HtlcManager` might seem that all of this htlc should be forwarded.
2. And also it seems for that it would be better if channel itself will not be aware of such term as `forwarding`. Instead channel just returns the htlcs and says - I treat them as fully included in both chains, and now you may do with them whatever you want.